bcdr/3.6/README.md

```diff
@@ -0,0 +1,11 @@
+# CP4WAIOps-BCDR
+
+- Code repo for automated backup and recovery of  functional components of CP4WAIOps on a new or existing cluster.
+- It uses OADP Operator  with s3 compliant object storage for storing state of the application.
+  - Internally it uses `velero`for cluster object backups
+  - It uses `restic` for volume backups
+- It can be deployed on any cluster post CP4AIOps deployment by using helm charts, or the supported artifacts
+- This backup/restore process follow the [design spec](https://github.ibm.com/katamari/architecture/blob/master/feature-specs/bcdr/bcdr.md)
```

Comment:
Do we have an external link for customers.

Reply:
considering we will use https://github.com/IBM/cp4waiops-samples/tree/main/bcdr for external users, we can remove the above item as it is only for internal use

Reply:
sorry this design spec shouldn't be part of documentation, we will update this

Reply:
@gyliu513 I have removed design spec link from the readme file, please check now.